src/Converter/ColumnNameConverter.cs

```diff
@@ -3,8 +3,8 @@
     #region Usings
     using NLog;
     using System;
+    using System.Globalization;
     using System.Windows.Data;
-    using System.Globalization; 
     #endregion
 
     public class ColumnNameConverter : IMultiValueConverter
```

Comment:
please make the Converter more save.
for example check if values.len >=  3 if you return an values[2]